talks/views.py

```diff
@@ -161,14 +162,20 @@ def get_context_data(self, **kwargs):
       
 
         if user.is_authenticated:
-            followTalkObj = FollowTalk.objects.filter(user=user, talk=Talk.objects.get(pk=self.kwargs['talk_pk']))
+            followTalkObj = FollowTalk.objects.filter(user=user, talk=post.talk)
+            
+            favoriteTalkObj = FavoriteTalk.objects.filter(user=user, talk=post.talk)
```

Comment:
These `camelCase` naming conventions are not `pythonic`. 
Switch your varaible name to use `lower_case_with_underscores`.
```python
follow_talk_obj = FollowTalk.objects.filter(user=user, talk=post.talk)
favortie_talk_obj = FavoriteTalk.objects.filter(user=user, talk=post.talk)
```